### PR TITLE
Quick fixes

### PR DESCRIPTION
**Pull request by ru-fu**
Two commits with original and update to make it easier to check the changes.

### CODE REVIEW

File: overview.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+Anbox Cloud provides a rich software stack to enable you to run Android in the cloud for all kinds of different uses cases. This page intends to give you an overview over the available variants and internal components which make Anbox Cloud.
+
+## Variants
+
+Anbox Cloud exists in two different variants which serve different purposes:
+
+**Anbox Cloud Appliance**
+
+The Anbox Cloud Appliance is a self-contained, single-machine deployment variant of Anbox Cloud. It uses a sensible standard configuration to hide the complexity and flexibility of Juju. In this way, the Anbox Cloud Appliance allows simple and fast development and prototyping.
```
Comment (ru-fu) [inline overview.md:9]: Updated this description. Are we okay with this wording?

File: overview.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+Anbox Cloud takes care of all management aspects and provides both a fully functional implementation and an integration model to support existing infrastructure and service implementations.
+
+The following figure gives an overview over the different components and their responsibility within the core stack of Anbox Cloud.
+
+![anbox-core-overview|690x398](upload://x0budFKybsbYLhiplNhF4R91Ght.png)
+
+At the heart of Anbox Cloud sits the **Anbox Management Service (AMS)**. **AMS** has the job to handle all aspects of the application and container lifecycle (including application and image updates) while ensuring high density, performance and fast container startup times.
+A developer or system administrator will manage **AMS** through the **command line interface (AMC)** or through custom-built tools interacting with the [**AMS REST API**](https://discourse.ubuntu.com/t/ams-rest-api-reference/17801).
+
+For example, a simple Android application testing service would provide a user-facing interface dealing with things like authentication and user management, and would communicate with the REST API to add applications or start and stop containers when a user asks to.
```
Comment (ru-fu) [inline overview.md:32]: This was "...communicate with the REST API to add applications, start and containers when a user asks to.". Is the update correct?

File: getting-started.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+## Ensure Images are Available
+
+As a next step you can check that **AMS** has synchronised all images from the Canonical hosted image server. You can list all synchronised images with the `amc image ls` command:
+
+```bash
++----------------------+-----------------------------+--------+----------+--------------+---------+
+| ID                   | NAME                        | STATUS | VERSIONS | ARCHITECTURE | DEFAULT |
++----------------------+-----------------------------+--------+----------+--------------+---------+
+| c4b4djkrorjohh948dfg | bionic:android11:arm64      | active | 1        | aarch64      | true    |
```
Comment (ru-fu) [inline getting-started.md:23]: I updated this table to be more recent, but noticed afterwards I see only the Arm images. Does this make sense, or should the table show the amd ones?

File: overview.md (hunk 1 above)
Comment (morphis) [reply overview.md:9]: Sounds good to me. Do we maybe want to link to Juju here to provide more context if needed?

File: overview.md (hunk 2 above)
Comment (morphis) [reply overview.md:32]: Sounds good to me

File: getting-started.md (hunk 3 above)
Comment (morphis) [reply getting-started.md:23]: Showing one of both is fine. We could even mix arm and x86 if we want as all these variants are possible.

**Comment by morphis** [comment]
@ru-fu Overall looks good to me. Feel free to merge and publish if you're ready.

**Pull request merged**